precode.sh

```diff
@@ -7,37 +7,75 @@
 #   dir3
 #       dir4
 
+mkdir -p ./task/dir1
```

Comment:
Можно и в одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```